components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1316,6 +1316,11 @@ fix_interpreter() {
     fi
 
     for t in ${targets}; do
+      if [[ ! -f $t ]]; then
+        warn "Ignoring non-file target: ${t}"
```

Comment:
This would be a common thing and not something we're actually try to call attention to. This might be better off being `debug` level instead of `warn` level